.github/workflows/archive_closed_jobs.js

```diff
@@ -1,14 +1,14 @@
 const _fs = require("fs/promises");
-const _path = require("path");
+const path = require("path");
```

Comment:
The `path` library as used here doesn't have any side effects, so we don't actually need to mock it.